app/templates/components/forms/login-form.hbs

```diff
@@ -25,7 +25,12 @@
           <div class="field">
             <div class="ui left icon input">
               <i class="lock icon"></i>
-              {{input type='password' name='password' placeholder=(t 'Password') value=password}}
+              {{input type='password' id="pwd" name='password' placeholder=(t 'Password') value=password}}
+              <div class="ui small basic icon buttons">
```

Comment:
Please don't use a button. Just use an icon. 

It is possible to have icons on both sides. -> https://semantic-ui.com/elements/input.html